css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  align-items: center;
+  margin: auto;
+  justify-content: center;
+  width: 900px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-half {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+}
+
+.column-one-third {
+  display: flex;
+  flex-direction: column;
```

Comment:
If you are applying `display: flex` to a column class, that usually means that the `column` should contain a `row` class element. That is the purpose of the `row` class, to make flex containers